docs/variables/user.md

```diff
@@ -4,137 +4,410 @@ id: user
+Returns a range of metadata around a (specified) user's channel that can be accessed via multiple different variable members for each unique piece of information.
 
-:::caution Timers are not supported.
+:::caution Timers are not supported!
 
-The variable will not return any value in a timer.
+Timers are executed on an interval by Fossabot itself and therefore this variable cannot be used inside of them.
+
+If your timer iterates through existing commands that use this variable, or if this variable is used in a timer response, Fossabot will return nothing in its place.
 
 :::
 
-## Example Output
+## Example Usage
+
+* A command with the name `!shoutout` and a response of the following:
+
+    ```
+    Shout out to $(user.display_name)! Go follow them at twitch.tv/$(user.login) - they were last seen playing $(user.game)!
+    ```
+
+    If triggered by `!shoutout aiden`, could return the following response:
 
-If you made a shoutout command:
+    ```
+    Shout out to Aiden! Go follow them at twitch.tv/aiden - they were last seen playing Just Chatting!
+    ```
 
-```
-Shout out to $(user.display_name)! Go follow them at twitch.tv/$(user.login) - they were last seen playing $(user.game)!
-```
+* A command with the name `!fossabot` and a response of the following:
 
-You could use `!shoutout aiden`, which would produce this output:
-```
-Shout out to Aiden! Go follow them at twitch.tv/aiden - they were last seen playing Just Chatting! 
-```
+    ```
+    Shout out to $(user.display_name fossabot)! Go follow them at twitch.tv/$(user.login fossabot) - they were last seen playing $(user.game fossabot)!
+    ```
 
-Or, you could hardcode a user, for example `!fossabot`, which would produce this output:
+    If triggered, could return the following response:
 
-```
-Shout out to $(user.display_name fossabot)! Go follow them at twitch.tv/$(user.login fossabot) - they were last seen playing $(user.game fossabot)!
-```
+    ```
+    Shout out to Fossabot! Go follow them at twitch.tv/fossabot - they were last seen playing <no game>!
+    ```
 
 ## $(user.accountage)
 
-Returns how long it has been since the user created their Twitch account.
+Returns the time that has passed since a user created their **Twitch account**.
+
+#### Parameters
+
+This variable takes **one** *optional* parameter that is a Twitch username of who to fetch the metadata of. Defaults to the sender if neither hardcoded nor provided at command execution, a user does not exist or is otherwise unavailable.
+
+#### Example Output
+
+* `$(user.accountage)`
 
-```
-4 years, 3 months and 2 days
-```
+    ```
+    6 years, 9 months, 10 days
+    ```
+
+* `$(user.accountage fossabot)`
+
+    ```
+    3 years, 6 months, 6 days
+    ```
+
+#### Error Output
+
+* In case an invalid username is provided, returns the following:
+
+    ```
+    [Error: Invalid username.]
+    ```
```

Comment:
maybe only needs to be stated once

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/18#discussion_r786586089).